app/factories/bulkrax/object_factory.rb

```diff
@@ -6,6 +6,14 @@ class ObjectFactory
     include Bulkrax::FileFactory
     include DynamicRecordLookup
 
+    # These are the attributes that we assume all "work type" classes (e.g. the given :klass) will
+    # have in addition to their specific attributes.
+    #
+    # @return [Array<Symbol>]
+    # @see #permitted_attributes
+    class_attribute :base_permitted_attributes,
```

Comment:
Breaking this out into a variable is a good idea. Is it configurable via bulkrax.rb? If so, should we add a commented out example in there?

And is this list of attributes only applied to works? In my local testing I ran into issues where Collections seemed to be getting `admin_set_id` called even though they didn't support it, so I ended up [excluding it via permitted_attributes.](https://github.com/UNC-Libraries/hy-c/blob/fae572c0ba4cd448e12560b5b060b48fa304d70d/app/overrides/factories/bulkrax/object_factory_override.rb#L68)

Reply:
@bbpennel This is one of those "quiet configuration points".  In the spec, I demonstrate that you can do the config on an instance by instance level.


But to make the change at the application level you'd likely add the following to your Bulkrax initializer:

```ruby
Bulkrax::ObjectFactory.base_permitted_attributes = %i[name title admin_set_id]
```

This is not a config that I want to consider a public API (and am updating the documentation accordingly)